File: cpp/Processor.cpp
#include <bits/stdc++.h>

using namespace std;

class Processor
{
private:
    string name;
    int price;

public:
    Processor()
    {
        this->name = "-";
        this->price = 0;
    }

    Processor(string name, int price)
    {
        this->name = name;
        this->price = price;
    }

    void setName(string name)
    {
        this->name = name;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    int getPrice()
    {
        return this->price;
    }

    string getName()
    {
        return this->name;
    }

    ~Processor(){};
};

File: python/Processor.py
class Processor():

    def __init__(self, name="-", price=0):
        self.__name = name
        self.__price = price

    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price
